tests/integration/components/live-participants-test.js

```diff
@@ -156,5 +158,6 @@ module('Integration | Component | live-participants', function (hooks) {
     />`);
     assert.dom('[data-test-sidebar-body-role-guest]').hasText('Guest (2)');
     assert.dom('[data-test-sidebar-user="3"]').hasText('Guest2');
+    assert.dom('.user').exists('data-test-icon');
```

Comment:
`data-test-icon` is not giving me idea what icon it is.
Loading or kickout ?

Reply:
kick out is an icon but Loading is not an icon, it's a div element
if you want any specific thing then please let me know clearly

Reply:
`data-test-kickout-icon` would be good here

Reply:
commented https://github.com/Real-Dev-Squad/website-www/pull/692#discussion_r1356520075